content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -49,7 +49,7 @@ this example.
 {{< /note >}}
 
 
-1. Configure the kubelet to be a service manager for etcd.
+1. Configure the kubelet to be a service manager for etcd. This must be done on all nodes.
 
```

Comment:
I don't think you do need to reconfigure the kubelet on every node; just on the hosts that are running the control plane (the page uses the word “node” in a misleading way, I think; in Kubernetes, _node_ usually means a computer that the scheduler can place Pods onto for execution). 

You could write this as:
```suggestion
1. Configure the kubelet to be a service manager for etcd.
  
   {{< note >}}You must do this on every host where etcd is running.{{< /note >}}
```

Reply:
Agreed.